src/libstd/dynamic_lib.rs

```diff
@@ -136,6 +133,7 @@ mod test {
     use libc;
     use old_path::Path;
```

Comment:
This import should be deleted, right? (allowing the below import to be `use path::Path`